android/README.md

```diff
@@ -491,3 +491,30 @@ Picture-in-Picture style scenario, in a rectangle too small to accommodate its
 Jitsi Meet SDK automatically enables (unless explicitly disabled by a
 `setPictureInPictureEnabled(false)` call) Android's native Picture-in-Picture
 mode iff the platform is supported i.e. Android >= Oreo.
+
+## Dropbox integration
+
+To setup the dropbox integration you need to do the following steps:
+1. Add the following section to the AndroidManifest and change `<APP_KEY>` to your dropbox app key:
+```
+<activity
+    android:name="com.dropbox.core.android.AuthActivity"
+    android:configChanges="orientation|keyboard"
+    android:launchMode="singleTask">
+  <intent-filter>
+
+    <!-- Change this to be db- followed by your app key -->
+    <data android:scheme="db-<APP_KEY>" />
+
+    <action android:name="android.intent.action.VIEW" />
+
+    <category android:name="android.intent.category.BROWSABLE" />
+    <category android:name="android.intent.category.DEFAULT" />
+  </intent-filter>
+</activity>
+```
+
+2. Add your dropbox app key to the strings.xml file:
+```
+<string name="dropbox_app_key"></string>
```

Comment:
@hristoterezov, the documentation on the Dropbox integration looks outdated because there steps are automatically performed by the Gradle build script now.

Reply:
They are not necessary for the Jitsi-meet app, but not for an app that integrates the SDK directly. @hristoterezov do I understand this correctly ?

Reply:
I was wrong, I'm sorry. Hristo explained in person: this is necessary for the SDK, not the app.